dept44-starter/src/main/java/se/sundsvall/dept44/configuration/LogbookConfiguration.java

```diff
@@ -69,6 +85,7 @@ Logbook logbook(final ObjectMapper objectMapper, final List<BodyFilter> bodyFilt
 				fileAttachmentFilter(),
 				binaryContentFilter()))
 			.bodyFilter(passwordFilter())
+			.bodyFilter(BodyFilters.truncate(maxBodySize))
```

Comment:
Only execute if `logbook.logs.maxBodySize` is set?

E.g. switch to Integer (instead of primitive int) and  allow the value to be null if not set. Then execute filter if value != null. Or something like that?